### ISSUE TITLE
Cannot find symbol valueOfLabel in generated Java class

### ISSUE THREAD

**Issue by longlho** [dotansimha/graphql-code-generator#6320]
**Describe the bug**
Generated Java class `enum` no longer has `valueOfLabel` and failed to compile

**To Reproduce**
https://codesandbox.io/s/hardcore-haslett-p41ng?file=/codegen.yml
<!--
#############
#############
🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 

PLEASE READ THE FOLLOWING: 
You can use this template to create a live sandbox of the issue: 
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template
Make sure to fork this template, and run `yarn generate` in the terminal. 
Please make sure the Codegen and plugins version under `package.json` matches yours. 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ ISSUES WITH A LIVE REPRODUCTION WILL BE ANSWERED QUICKLY ~~~~~~~
~~~~~~ ISSUES WITHOUT REPRODUCTION MIGHT GET CLOSED WITHOUT RESPONSE ~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#############
#############
-->

1. My GraphQL schema:

```graphql
type Query {
    user(id: ID!): User!
}

type Mutation {
    change(test: Input!): Resp
}

type User {
    id: ID!
    username: String!
    email: String!
}

input Input {
    test: Test!
}

type Resp {
    test: Test!
}

enum Test {
    Foo
    Bar
}
```

2. My GraphQL operations:

```graphql
query user {
    user(id: 1) {
        id
        username
        email
    }
}

```

3. My `codegen.yml` config file:

```yml
schema: schema.graphql
documents: document.graphql
generates:
  Types.java:
    plugins:
      - java

```

**Expected behavior**
`enum Test` should have had `valueOfLabel` method declared

**Environment:**

- OS: Mac
- package.json:
```
"@graphql-codegen/add": "2.0.2",
"@graphql-codegen/cli": "1.21.7",
"@graphql-codegen/java": "2.0.4",
"graphql": "15.5.0"
```
- NodeJS: 16

**Comment by longlho** [comment]
looks like https://github.com/dotansimha/graphql-code-generator/commit/a290a4ce9a68654f370499db3fa6eb49bc2bef82 removed it

**Comment by dotansimha** [comment]
Fixed in `2.0.0` of the Java plugins

**Comment by tzachs** [comment]
@dotansimha / @longlho sorry for that.
